src/tests/ftest/cart/util/cart_utils.py

```diff
@@ -222,6 +225,9 @@ def get_env(self):
         if ofi_share_addr is not None:
             env += " -x CRT_CTX_SHARE_ADDR={!s}".format(ofi_share_addr)
 
+        if server_continuous is not None:
+            env += " -x CRT_TEST_CONT={!s}".format(server_continuous)
```

Comment:
is this needed? why do we need to export CRT_TEST_CONT on all nodes; we only care about --continuous flag being st on orterun

Reply:
Not necessary.  Removed.